### ISSUE TITLE
【EESS】New submissions for Wed, 20 Sep 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#663]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
### Assessing the capacity of a denoising diffusion probabilistic model to  reproduce spatial context
 - **Authors:** Rucha Deshpande, Muzaffer Özbey, Hua Li, Mark A. Anastasio, Frank J. Brooks
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2309.10817
 - **Pdf link:** https://arxiv.org/pdf/2309.10817
 - **Abstract**
 Diffusion models have emerged as a popular family of deep generative models (DGMs). In the literature, it has been claimed that one class of diffusion models -- denoising diffusion probabilistic models (DDPMs) -- demonstrate superior image synthesis performance as compared to generative adversarial networks (GANs). To date, these claims have been evaluated using either ensemble-based methods designed for natural images, or conventional measures of image quality such as structural similarity. However, there remains an important need to understand the extent to which DDPMs can reliably learn medical imaging domain-relevant information, which is referred to as `spatial context' in this work. To address this, a systematic assessment of the ability of DDPMs to learn spatial context relevant to medical imaging applications is reported for the first time. A key aspect of the studies is the use of stochastic context models (SCMs) to produce training data. In this way, the ability of the DDPMs to reliably reproduce spatial context can be quantitatively assessed by use of post-hoc image analyses. Error-rates in DDPM-generated ensembles are reported, and compared to those corresponding to a modern GAN. The studies reveal new and important insights regarding the capacity of DDPMs to learn spatial context. Notably, the results demonstrate that DDPMs hold significant capacity for generating contextually correct images that are `interpolated' between training samples, which may benefit data-augmentation tasks in ways that GANs cannot.
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
### Incorporating Ultrasound Tongue Images for Audio-Visual Speech  Enhancement
 - **Authors:** Rui-Chen Zheng, Yang Ai, Zhen-Hua Ling
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2309.10455
 - **Pdf link:** https://arxiv.org/pdf/2309.10455
 - **Abstract**
 Audio-visual speech enhancement (AV-SE) aims to enhance degraded speech along with extra visual information such as lip videos, and has been shown to be more effective than audio-only speech enhancement. This paper proposes the incorporation of ultrasound tongue images to improve the performance of lip-based AV-SE systems further. To address the challenge of acquiring ultrasound tongue images during inference, we first propose to employ knowledge distillation during training to investigate the feasibility of leveraging tongue-related information without directly inputting ultrasound tongue images. Specifically, we guide an audio-lip speech enhancement student model to learn from a pre-trained audio-lip-tongue speech enhancement teacher model, thus transferring tongue-related knowledge. To better model the alignment between the lip and tongue modalities, we further propose the introduction of a lip-tongue key-value memory network into the AV-SE model. This network enables the retrieval of tongue features based on readily available lip features, thereby assisting the subsequent speech enhancement task. Experimental results demonstrate that both methods significantly improve the quality and intelligibility of the enhanced speech compared to traditional lip-based AV-SE baselines. Moreover, both proposed methods exhibit strong generalization performance on unseen speakers and in the presence of unseen noises. Furthermore, phone error rate (PER) analysis of automatic speech recognition (ASR) reveals that while all phonemes benefit from introducing ultrasound tongue images, palatal and velar consonants benefit most.
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### Bearing and Distance Formation Control of Rigid Bodies in SE(3) with  Bearing and Distance Constraints
 - **Authors:** Sara Mansourinasab, Mahdi Sojoodi, Seyed Reza Moghadasi
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.10183
 - **Pdf link:** https://arxiv.org/pdf/2309.10183
 - **Abstract**
 Rigidity of the interaction graph is a fundamental condition for achieving the desired formation which can be defined in terms of distance or bearing constraints between agents. In this paper, for reaching a unique formation with the same scaling and orientation as the target formation, both distance and bearing constraints are considered for defining the desired formation. Besides, both distance and bearing measurements are also available. Each agent is able to gather the measurements with respect to other agents in its own body frame. So, the agents are coordinated-free concerning a global reference frame. On the other hand, the framework is embedded in SE(3). The control signal is designed based on a gradient descent method by introducing a cost function. Firstly, the formation problem is considered for bearing-only constraints in SE(3) configuration. Then, the formation control is expressed for the general case of both bearing and distance constraints. Furthermore, the essential conditions that guarantee reaching the desired formation are discussed. Finally, the validity of the proposed formation control is verified by numerical simulations.
### ProtoKD: Learning from Extremely Scarce Data for Parasite Ova  Recognition
 - **Authors:** Shubham Trehan, Udhav Ramachandran, Ruth Scimeca, Sathyanarayanan N. Aakur
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2309.10210
 - **Pdf link:** https://arxiv.org/pdf/2309.10210
 - **Abstract**
 Developing reliable computational frameworks for early parasite detection, particularly at the ova (or egg) stage is crucial for advancing healthcare and effectively managing potential public health crises. While deep learning has significantly assisted human workers in various tasks, its application and diagnostics has been constrained by the need for extensive datasets. The ability to learn from an extremely scarce training dataset, i.e., when fewer than 5 examples per class are present, is essential for scaling deep learning models in biomedical applications where large-scale data collection and annotation can be expensive or not possible (in case of novel or unknown infectious agents). In this study, we introduce ProtoKD, one of the first approaches to tackle the problem of multi-class parasitic ova recognition using extremely scarce data. Combining the principles of prototypical networks and self-distillation, we can learn robust representations from only one sample per class. Furthermore, we establish a new benchmark to drive research in this critical direction and validate that the proposed ProtoKD framework achieves state-of-the-art performance. Additionally, we evaluate the framework's generalizability to other downstream tasks by assessing its performance on a large-scale taxonomic profiling task based on metagenomes sequenced from real-world clinical data.
### Learning Dynamic MRI Reconstruction with Convolutional Network Assisted  Reconstruction Swin Transformer
 - **Authors:** Di Xu, Hengjie Liu, Dan Ruan, Ke Sheng
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2309.10227
 - **Pdf link:** https://arxiv.org/pdf/2309.10227
 - **Abstract**
 Dynamic magnetic resonance imaging (DMRI) is an effective imaging tool for diagnosis tasks that require motion tracking of a certain anatomy. To speed up DMRI acquisition, k-space measurements are commonly undersampled along spatial or spatial-temporal domains. The difficulty of recovering useful information increases with increasing undersampling ratios. Compress sensing was invented for this purpose and has become the most popular method until deep learning (DL) based DMRI reconstruction methods emerged in the past decade. Nevertheless, existing DL networks are still limited in long-range sequential dependency understanding and computational efficiency and are not fully automated. Considering the success of Transformers positional embedding and "swin window" self-attention mechanism in the vision community, especially natural video understanding, we hereby propose a novel architecture named Reconstruction Swin Transformer (RST) for 4D MRI. RST inherits the backbone design of the Video Swin Transformer with a novel reconstruction head introduced to restore pixel-wise intensity. A convolution network called SADXNet is used for rapid initialization of 2D MR frames before RST learning to effectively reduce the model complexity, GPU hardware demand, and training time. Experimental results in the cardiac 4D MR dataset further substantiate the superiority of RST, achieving the lowest RMSE of 0.0286 +/- 0.0199 and 1 - SSIM of 0.0872 +/- 0.0783 on 9 times accelerated validation sequences.
### Safe Control Design through Risk-Tunable Control Barrier Functions
 - **Authors:** Vipul K. Sharma, S. Sivaranjani
 - **Subjects:** Systems and Control (eess.SY); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2309.10251
 - **Pdf link:** https://arxiv.org/pdf/2309.10251
 - **Abstract**
 We consider the problem of designing controllers to guarantee safety in a class of nonlinear systems under uncertainties in the system dynamics and/or the environment. We define a class of uncertain control barrier functions (CBFs), and formulate the safe control design problem as a chance-constrained optimization problem with uncertain CBF constraints. We leverage the scenario approach for chance constrained optimization to develop a risk-tunable control design that provably guarantees the satisfaction of CBF safety constraints up to a user-defined probabilistic risk bound, and provides a trade-off between the sample complexity and risk tolerance. We demonstrate the performance of this approach through simulations on a quadcopter navigation problem with obstacle avoidance constraints.
### Self-Supervised Super-Resolution Approach for Isotropic Reconstruction  of 3D Electron Microscopy Images from Anisotropic Acquisition
 - **Authors:** Mohammad Khateri, Morteza Ghahremani, Alejandra Sierra, Jussi Tohka
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2309.10646
 - **Pdf link:** https://arxiv.org/pdf/2309.10646
 - **Abstract**
 Three-dimensional electron microscopy (3DEM) is an essential technique to investigate volumetric tissue ultra-structure. Due to technical limitations and high imaging costs, samples are often imaged anisotropically, where resolution in the axial direction ($z$) is lower than in the lateral directions $(x,y)$. This anisotropy 3DEM can hamper subsequent analysis and visualization tasks. To overcome this limitation, we propose a novel deep-learning (DL)-based self-supervised super-resolution approach that computationally reconstructs isotropic 3DEM from the anisotropic acquisition. The proposed DL-based framework is built upon the U-shape architecture incorporating vision-transformer (ViT) blocks, enabling high-capability learning of local and global multi-scale image dependencies. To train the tailored network, we employ a self-supervised approach. Specifically, we generate pairs of anisotropic and isotropic training datasets from the given anisotropic 3DEM data. By feeding the given anisotropic 3DEM dataset in the trained network through our proposed framework, the isotropic 3DEM is obtained. Importantly, this isotropic reconstruction approach relies solely on the given anisotropic 3DEM dataset and does not require pairs of co-registered anisotropic and isotropic 3DEM training datasets. To evaluate the effectiveness of the proposed method, we conducted experiments using three 3DEM datasets acquired from brain. The experimental results demonstrated that our proposed framework could successfully reconstruct isotropic 3DEM from the anisotropic acquisition.
### Assessing the capacity of a denoising diffusion probabilistic model to  reproduce spatial context
 - **Authors:** Rucha Deshpande, Muzaffer Özbey, Hua Li, Mark A. Anastasio, Frank J. Brooks
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG); Machine Learning (stat.ML)
 - **Arxiv link:** https://arxiv.org/abs/2309.10817
 - **Pdf link:** https://arxiv.org/pdf/2309.10817
 - **Abstract**
 Diffusion models have emerged as a popular family of deep generative models (DGMs). In the literature, it has been claimed that one class of diffusion models -- denoising diffusion probabilistic models (DDPMs) -- demonstrate superior image synthesis performance as compared to generative adversarial networks (GANs). To date, these claims have been evaluated using either ensemble-based methods designed for natural images, or conventional measures of image quality such as structural similarity. However, there remains an important need to understand the extent to which DDPMs can reliably learn medical imaging domain-relevant information, which is referred to as `spatial context' in this work. To address this, a systematic assessment of the ability of DDPMs to learn spatial context relevant to medical imaging applications is reported for the first time. A key aspect of the studies is the use of stochastic context models (SCMs) to produce training data. In this way, the ability of the DDPMs to reliably reproduce spatial context can be quantitatively assessed by use of post-hoc image analyses. Error-rates in DDPM-generated ensembles are reported, and compared to those corresponding to a modern GAN. The studies reveal new and important insights regarding the capacity of DDPMs to learn spatial context. Notably, the results demonstrate that DDPMs hold significant capacity for generating contextually correct images that are `interpolated' between training samples, which may benefit data-augmentation tasks in ways that GANs cannot.
